src/Orleans.Core/Statistics/ClientTableStatistics.cs

```diff
@@ -1,142 +0,0 @@
-#define LOG_MEMORY_PERF_COUNTERS 
-using System;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
-using Orleans.Statistics;
-
-namespace Orleans.Runtime
-{
-    internal class ClientTableStatistics : IClientPerformanceMetrics, IDisposable
```

Comment:
why is this removed? isn't this statistics, not metrics?

Reply:
The class name is statistics, but the interface is metrics. This is only used for snapshot metrics